docs/source/index.rst

```diff
@@ -21,3 +22,4 @@ Cleanlab Studio automatically finds and fixes errors in your dataset to turn unr
 This package provides a Python API and CLI to interact with the Cleanlab Studio app.
 
 .. include:: /quickstart/index.rst
+.. include:: /tutorials/index.rst
```

Comment:
I don't think we want to display this (this is just a `Tutorials` header w/ nothing underneath) on our index page. just adding to the toctree should be sufficient to have it appear in the sidebar